api/context.go

```diff
@@ -107,6 +108,7 @@ func (h handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	}
 
 	c := &Context{}
+	c.App = app.Global()
```

Comment:
The app won't be created on every request. It will be injected into the HTTP server at a higher level.